ooniprobe/Images.xcassets/settings/about_ooni.imageset/Contents.json

```diff
@@ -8,5 +8,8 @@
   "info" : {
     "version" : 1,
     "author" : "xcode"
+  },
+  "properties" : {
+    "template-rendering-intent" : "template"
```

Comment:
Why is this change appearing here?

Reply:
So, wait, why did you resolve the conversation without responding?

Reply:
Maybe I did by mistake. This is a change that should be here. it's an image that changed

Reply:
After checking another branch I noticed is an useless change as this image is used in setting and it has already colors.

Reply:
Ok, thanks for clarifying.

Reply:
My bad, this is needed to keep that image attributes in "sync" with others that are all considered "template".
Template is an image that can be colored used setTintColor